android/src/main/java/com/mapbox/mapboxgl/MapboxMapController.java

```diff
@@ -445,7 +546,9 @@ public void setMyLocationEnabled(boolean myLocationEnabled) {
   }
 
   private void updateMyLocationEnabled() {
-    //throw new UnsupportedOperationException("updateMyLocationEnabled")
+    // if (locationComponent != null)  {
+    //   locationComponent.setLocationComponentEnabled(this.myLocationEnabled);
+    // }
```

Comment:
This code is not used and now that we have `enableLocationComponent`, can we remove it?

Reply:
We may need some code here in the future, if we start with myLocationEnabled=false and turn it to true later.